### ISSUE TITLE
Visual feedback

### ISSUE THREAD

**Issue by Const4ntFlux** [dbkynd/controlcast#8]
Is it possible to enable the Launchpad to give visual feedback for what is happening/has been pressed?
It would require an option in the Edit Key menu for 'Triggered' for example where you can define the color for when a button has been pressed and it also would be necessary to create logical groups of buttons.

Example:
Six buttons for six scenes. x(group)1-6
All buttons colors configured:
Inactive: Red
Active: off
Triggered: Green

User presses button x1 to go to scene 1
Visual feedback: button x1 is green all other buttons x2-6  red

User presses button x2 to go to scene 2
Visual feedback: button x2 is green all other buttons x1 and x3-6  red

This of course would be different for the audio buttons:
I'd suggest a triggered color option for the audio buttons to define a state to indicate the corresponding audio file is being played.

In respect to scene switching: I understand that probably Controlcast can't 'watch' what scene is active (it would be awesome if OBS/XSplit would provide a way to do so though), so if one would switch to different scenes with the mouse, the visual indicators of the Launchpad would differ from what is actually happening in the streaming application. This is a big problem, but if noted in the readme and a 'neutral' state button would be integrated in Controlcast, it could be managed.

**Comment by dbkynd** [comment]
Yeah, I had thought about grouping keys together for the scenes, but hadn't decided on the method I want to use.

I thought maybe another section in the Edit Key area that lets you change the color state of any/all of the other keys on press. But wasn't sure how I wanted to format that.

**Comment by Zelakto** [comment]
1+

**Comment by cptnchoco** [comment]
+1 too <3

**Comment by Zackery** [comment]
Any progress on this? I feel like this is the one thing that's really preventing me from using this. 😢
